example_code/applications/englisg_auction/Cargo.toml

```diff
@@ -1,20 +1,27 @@
 [package]
 name = "stylus-auction-example"
-version = "0.1.5"
+version = "0.1.7"
 edition = "2021"
-keywords = ["arbitrum", "ethereum", "stylus", "alloy", "english auction"]
-description = "Stylus english auction example"
+license = "MIT OR Apache-2.0"
+homepage = "https://github.com/OffchainLabs/stylus-hello-world"
+repository = "https://github.com/OffchainLabs/stylus-hello-world"
+keywords = ["arbitrum", "ethereum", "stylus", "alloy"]
+description = "Stylus hello world example"
```

Comment:
Any reason why we still keep `Stylus hello world example` as description here?

Reply:
Oh nice catch! I forgot to remove them!